x/oracle/genesis.go

```diff
@@ -40,16 +40,30 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	for _, elem := range genState.StakerInfosAssets {
 		k.SetStakerInfos(ctx, elem.AssetId, elem.StakerInfos)
 	}
+	// set validatorReportInfos
+	for _, elem := range genState.ReportInfos {
+		k.SetValidatorReportInfo(ctx, elem.Address, elem)
+	}
+	// set vlidatorMissedRounds
```

Comment:
typo: validatorMissedRounds

Reply:
@coderabbitai  fix this typo ：vlidatorMissedRounds

Reply:
@coderabbitai  fix this typo ：vlidatorMissedRounds and commit the fix